code/game/objects/items/weapons/manuals.dm

```diff
@@ -1183,3 +1183,891 @@
+                    <li>The Clown is not permitted to remove their Clown Shoes or Clown Mask. Exception is made if removing them is truly necessary for the sake of their clowning performance (such as being a satire of bad clowns);</li>
+                    <li>The Clown is not permitted to hold anything but water in their Sunflower;</li>
+                    <li>The Clown is not permitted to use Space Lube on anything. Exception is made during major emergencies involving hostile humanoids, whereby use of Space Lube may be condoned to help the crew;</li>
+                    <li>The Clown must legitimately attempt to be funny and/or entertaining at least once every fifteen (15) minutes. A simple pun will suffice. Continuously slipping people for no reason does not constitute humour. The joke is supposed to be funny for everyone;</li>
+                    <li>The Clown is permitted to, and freely exempt from any consequences of, performing any harmless prank that does not directly conflict with the above Guidelines</li>
+				</ol>
+                </div>
+                <br><br>
+
+                <h1><a name="mime"><B>Mime</B></a></h1><BR>
+				<ol>
+					<li>The Mime is not permitted to talk, under any circumstance whatsoever. A Mime who breaks the Vow of Silence is to be stripped of their rank post haste;</li>
+                    <li>The Mime is permitted to use written words to communicate, either via paper or PDA, but are discouraged from automatically resorting to it when miming will suffice;</li>
+                    <li>The Mime must actually mime something at least once every thirty (30) minutes. Standing against an invisible wall will suffice.</li>
+                </ol>
+                <br><br>
+
+                <h1><a name="chap"><B>Chaplain</B></a></h1><BR>
+				<ol>
+					<li>The Chaplain is not permitted to execute Bible Healing without consent, unless the person in question is in Critical Condition and there are no doctors, as doing so incurs the risk of causing brain damage;</li>
+                    <li>The Chaplain may not draw the Null Rod or Holy Sword on any personnel. Using these items on any personnel is grounds to have these items confiscated, unless there is a clear and present danger to their life;</li>
+                    <li>The Chaplain may not actively discriminate against any personnel on the grounds that it is a religious tenet of their particular faith;</li>
+                    <li>The Chaplain may not perform funerals for any personnel that have since been cloned;</li>
+                    <li>The Chaplain may, however, freely conduct funerals for non-cloneable personnel. All funerals must be concluded with the use of the Mass Driver or Crematorium.</li>

```

Comment:
even if it's a minor thing, not really comfortable make that kind of changes by my own, so, @TullyBurnalot